### ISSUE TITLE
Translating emails in 1.2

### ISSUE THREAD

**Issue by MaggieCabrera** [intrip/laravel-authentication-acl#59]
I have seen that in 1.2 you made a messages.php file with all the messages there to translate. That is perfect, but it's still in the vendor folder, how should I extend it so that it won't be lost when I make an update?

**Comment by MaggieCabrera** [comment]
Also, inside ReminderService.php there is also this line: 

```
 private function prepareResetPasswordLink($token, $to)
     {
         $this->body = link_to_action("Jacopo\\Authentication\\Controllers\\AuthController@getChangePassword","Click here to change your password.", ["email"=> $to, "token"=> $token] );
    }
```

Click here to change your password cannot be translated from messages

**Comment by intrip** [comment]
You need to do php artisan config:publish and change the config in app/config/packages/jacopo/laravel-authentication-acl/messages.php. For this string i'll push a fix when i can

**Comment by intrip** [comment]
I've pushed the fix in branch 1.2 use composer dev-1.2 to try that. Give me some feedback please.
